carousel/docs/getting_started.rst

```diff
@@ -124,11 +124,21 @@ Therefore models can be created three different ways.
         """
         Layers specified as class attributes
         """
-        data = [(MyModelData, {'filename': 'data.json'}), ...]
-        outputs = [MyModelOutputs, ...]
-        formulas = [MyModelFormulas, ...]
-        calculations = [MyModelCalculations, ...]
-        simulations = [MyModelSimulations]
+        data = ModelParameter(
+            layer='Data', sources=[(MyModelData, {'filename': 'data.json'}), ...]
+        )
+        outputs = ModelParameter(
+            layer='Outputs', sources=[MyModelOutputs, ...]
+        )
+        formulas = ModelParameter(
+            layer='Formulas', [MyModelFormulas, ...]
```

Comment:
Python calls can't have keyword arguments _precede_ positional args, so this would raise an error

>SyntaxError: non-keyword arg after keyword arg

see [keyword arguments in Python docs](https://docs.python.org/2/tutorial/controlflow.html#keyword-arguments)